lib/fluent/plugin/s3_compressor_gzip_command.rb

```diff
@@ -4,6 +4,7 @@ class GzipCommandCompressor < Compressor
       S3Output.register_compressor('gzip_command', self)
 
       config_param :command_parameter, :string, :default => ''
+      config_param :customer_tmp_dir, :string, :default => Dir.tmpdir
```

Comment:
No, I use same `tmp_dir` for  output and compressor. I test add `customer_tmp_dir` only in `out_s3.rb`, it does not work in `s3_compressor_gzip_command.rb`, so add duplicative code.

Is there a way get `customer_tmp_dir` config in `s3_compressor_gzip_command.rb`? Thank you.